Sources/Schema.swift

```
@@ -13,6 +13,7 @@ public enum Schema {
     case boolean(metadata: Metadata)
     case file(metadata: Metadata)
     case any(metadata: Metadata)
+    case unresolvedReference(String)
```

I don't want to expose unresolved references in the public API - I would rather we throw an UnresolvedReference error instead.